testsuite/python/lb.py

```diff
@@ -656,7 +656,7 @@ def test_viscous_coupling_rounding(self):
         self.system.lb = lbf
         self.system.thermostat.set_lb(LB_fluid=lbf, seed=3, gamma=self.gamma)
         p = self.system.part.add(pos=[-1E-30] * 3, v=[-1, 0, 0])
-        self.system.integrator.run(1)
+        self.system.integrator.run(2)
         for _ in range(20):
```

Comment:
Here (and in some other places), why do we need one extra time step? In the integrate_pull() function in LBWalberlaImpl.hpp ResetForces is the first call, so shouldn't the force be applied immediately?